content/shmem_broadcast.tex

```diff
@@ -75,7 +75,7 @@
     
     Upon return from a broadcast routine, the following are true for the local
     \ac{PE}: If the current \ac{PE} is not the root \ac{PE}, the \dest{} data object
-    is updated. The \source{} data object may be safely reused.  
+    is updated. The \source{} data object on the root \ac{PE} may be safely reused.  
```

Comment:
This introduces a semantic change. Please amend the commit.

Reply:
I meant it as a clarification since a non-root PE's source data object is not involved in this call.